src/components/DataElement/Recent.jsx

```diff
@@ -66,7 +68,7 @@ export default function Recent({isLoading, data}) {
                   {count}
                 </Paragraph4>
               </StyledCell>
-              <StyledCell>
+              <StyledCell onClick={() => setClickedCity(city)} style={{cursor: 'pointer'}}>
                 <Paragraph4
                   margin={0}
                 >
```

Comment:
I would probably move the functionality lower to `Paragraph4` component or even maybe create `StyledLink`. Also I think an user shouldn't be able to click on "Brak szczegółów". And one more thing regarding `style` prop - if it is used like this it skips Styletron engine and is rendered directly to html style attribute, which isn't what we want. Preferably I would use `$style` prop on `Paragraph4` component, but if you use `StyledLink` there won't be need for that.

Reply:
I think that the user should be able to click "Brak szczegółów" as long as it takes him to initial view. That hasn't any negative side effects and - in my opinion - is a lot more user friendly.